crates/nu-parser/src/parse_keywords.rs

```diff
@@ -2286,6 +2286,28 @@ pub fn parse_let(
                             parse_var_with_opt_type(working_set, &spans[1..(span.0)], &mut idx);
                         error = error.or(err);
 
+                        let var_name =
+                            String::from_utf8_lossy(working_set.get_span_contents(spans[1]))
+                                .to_string();
```

Comment:
Sorry, I have no idea about it, can you please explain why we need to get_span_contents from `spans[1]`, but not from `lvalue`'s span?

Reply:
Both are the same afaik, but I guess using `lvalue`'s span will make more sense.